### PR TITLE
remove the word simply from a couple challenges

### PR DESCRIPTION
**Pull request by QuincyLarson**
This is to be consistent with the observation that "simply" is a presumptuous word that generally doesn't belong in tutorials or lessons. I'm writing a big Medium article about this.

### CODE REVIEW

File: public/c7979f1c69945c59.html [hunk 1]
```diff
@@ -1 +0,0 @@
-simply-hired-verification: c7979f1c69945c59.html
```
Comment (BerkeleyTrue) [inline public/c7979f1c69945c59.html]: What's this file?

**Comment by atjonathan** [comment]
@QuincyLarson, please remove `public/c7979f1c69945c59.html` and we should be good to go :+1:

File: public/c7979f1c69945c59.html (hunk 1 above)
Comment (QuincyLarson) [inline public/c7979f1c69945c59.html]: This is from that day we tried Simply Hired and made about $0.75.

**Comment by QuincyLarson** [comment]
@atjonathan not sure what you mean - didn't my commit remove public/c7979f1c69945c59.html already?

**Comment by atjonathan** [comment]
@QuincyLarson you removed the contents of the file, but not the file itself :wink:

**Comment by QuincyLarson** [comment]
@atjonathan I checked the repo in both Webstorm and in my terminal, and that file no longer existed in either. Unless you are still showing otherwise, please QA and merge this PR.

**Comment by atjonathan** [comment]
### **LGTM :+1:**
It still seems to show on github as not deleted, but the file isn't there locally. Thanks 👍

**Pull request merged**